trim.production.js

```diff
@@ -31,16 +31,24 @@ function createRecord (title, container, extension, fileData, alternativeContain
     throw new Error('alternativeContainers must be a list of strings: ' + alternativeContainers)
   }
 
+  var body = {
+    Title: title,
+    Container: container,
+    RecordExtension: extension,
+    AlternativeContainers: alternativeContainers,
+    Record: fileData
+  }
+
+  body = R.evolve({
+    Title: R.ifElse(R.isNilOrEmpty, R.always('Untitled Document'), R.replace(/:/g, '-')),
+    RecordExtension: R.replace(/./g, ''),
```

Comment:
:scream: I obviously did not actually test that one. I was in a rush and put it in last second and thought "this looks right. it will be fine"